policies/templates/gcp_gke_container_optimized_os.yaml

```diff
@@ -57,11 +57,13 @@ spec:
             	constraint := input.constraint
             	asset := input.asset
             	asset.asset_type == "container.googleapis.com/Cluster"
+            	lib.get_constraint_params(constraint, params)
+            	containerd_allowed = lib.get_default(params, "cos_containerd_allowed", false)
```

Comment:
The new param should be added to the openAPI spec for the template (line 29).

Reply:
Added. Thanks!